### PR TITLE
{lib}[GCCcore/10.3.0] mimalloc v1.7.2

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/b37d1f429215ab491a71a08a4881472f for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/cde21c9bda151d2c31308326f09c4cc8 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/421a18da8f4cd8018d6dc47739967589 for a full test report.

File: easybuild/easyconfigs/m/mimalloc/mimalloc-1.7.2-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+sanity_check_paths = {
+    'files': ['lib/mimalloc-%(version_major_minor)s/libmimalloc.a', 'lib/libmimalloc.%s' % SHLIB_EXT],
+    'dirs': ['include/mimalloc-%(version_major_minor)s'],
+}
+
+runtest = 'test'
+
+# jemalloc can be used via $LD_PRELOAD, but we don't enable this by
+# default, you need to opt-in to it
```
Comment (lexming) [inline easybuild/easyconfigs/m/mimalloc/mimalloc-1.7.2-GCCcore-10.3.0.eb:29]: I was very puzzled by the mention to `jemalloc` here, I would use the same text found in mimalloc description
```suggestion
# mimalloc is a drop-in replacement for malloc and can be used in other programs without code changes,
# but we don't enable this by default, you need to opt-in by setting $LD_PRELOAD as follows
```

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/mimalloc/mimalloc-1.7.2-GCCcore-10.3.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/m/mimalloc/mimalloc-1.7.2-GCCcore-10.3.0.eb]: copy pasted from jemalloc and i forgot to replace the name

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/3cc59f8657b81c70f4e102a8fb9b4c1c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6c4feb39e7e9727521c67f7e7f134b2f for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Micket !

**Pull request merged**